docusearch/templates/search/search.html

```diff
@@ -40,7 +40,7 @@
         {% for result in page.object_list %}
             <li>
                 <div class="agencies--name">
-                    <h3><a href="{{ result.object.get_absolute_url() }}">{{result.object.title}}</a></h3>
+                    <h3><a href="{{ result.object.get_absolute_url()}}/?q={{request.GET.q}}">{{result.object.title}} ({{result.object.date}})</a></h3>
```

Comment:
Occasionally, the date is missing, we could display something a little smother like ({{result.object.date or "Missing Date"}})